src/resources/views/taxon/_tree.blade.php

```diff
@@ -12,19 +12,21 @@ class="collapse-toggler-heading">
         @endif
 
         @can('edit taxons')
-            <a href="{{ route('vanilo.admin.taxon.edit', [$taxonomy, $taxon]) }}">{{ $taxon->name }}</a>
+            <x-appshell::button :href="route('vanilo.admin.taxon.edit', [$taxonomy, $taxon])" variant="link">{{ $taxon->name }}</x-appshell::button>
```

Comment:
No, we don't convert every `a href` to button, only those having the look and feel of a button, ie. those having the `btn` class